terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/codelist/CodeListInterceptorTest.java

```diff
@@ -402,187 +394,4 @@ public void testAfterPropertiesSet_applicationContext_is_null() throws Exception
-        codeList.setRows(rows);
-
-        // do test.
-        Map<String, String> actualMap = testTarget.getLocalizedCodeMap(codeList,
-                Locale.ENGLISH);
-
-        // do assert.
-        assertThat(actualMap, is(enMap));
-
-    }
-
-    /**
-     * [getLocalizedCodeListMap] Case of fall back locale is null.
-     * <p>
-     * [Expected Result]
-     * <ol>
-     * <li>get empty map.(get localized code list map by locale.)</li>
-     * </ol>
-     * </p>
-     * @throws Exception
-     */
-    @Test
-    public void testGetLocalizedCodeListMap_fall_back_locale_is_null() {
-
-        // do setup.
-        testTarget.setFallbackTo(null);
-
-        SimpleI18nCodeList codeList = new SimpleI18nCodeList();
-        Map<Locale, Map<String, String>> rows = Maps.newLinkedHashMap();
-        Map<String, String> enMap = Collections.emptyMap();
-        rows.put(Locale.ENGLISH, enMap);
-        codeList.setRows(rows);
-
-        // do test.
-        Map<String, String> actualMap = testTarget.getLocalizedCodeMap(codeList,
-                Locale.ENGLISH);
-
-        // do assert.
-        assertThat(actualMap, is(enMap));
-
-    }
-
-    /**
-     * [getLocalizedCodeListMap] Case of not define.
-     * <p>
-     * [Expected Result]
-     * <ol>
-     * <li>get empty map.</li>
-     * <li>output warn log.</li>
-     * </ol>
-     * </p>
-     * @throws Exception
-     */
-    @Test
-    public void testGetLocalizedCodeListMap_not_define() {
-
-        // do setup.
-        testTarget.setFallbackTo(Locale.ENGLISH);
-
-        SimpleI18nCodeList codeList = new SimpleI18nCodeList();
-        Map<Locale, Map<String, String>> rows = Maps.newLinkedHashMap();
-        codeList.setRows(rows);
-
-        // do test.
-        Map<String, String> actualMap = testTarget.getLocalizedCodeMap(codeList,
-                Locale.JAPANESE);
-
-        // do assert.
-        assertThat(actualMap.isEmpty(), is(true));
-
-        verifyLogging("could not fall back to 'en'.", Level.WARN);
-
-    }
-
-    /**
-     * verify logging.
-     * @param expectedLogMessage expected log message.
-     * @param expectedLogLevel expected log level.
-     */
-    private void verifyLogging(final String expectedLogMessage,
-            final Level expectedLogLevel) {
-        verify(mockAppender).doAppend(argThat(
-                new ArgumentMatcher<LoggingEvent>() {
-                    @Override
-                    public boolean matches(LoggingEvent argument) {
-                        return argument.getFormattedMessage().equals(
-                                expectedLogMessage);
-                    }
-                }));
-        verify(mockAppender).doAppend(argThat(
-                new ArgumentMatcher<LoggingEvent>() {
-                    @Override
-                    public boolean matches(LoggingEvent argument) {
-                        return expectedLogLevel.equals(argument.getLevel());
-                    }
-                }));
-
-    }
-
```

Comment:
> ※JUnitは@hirakuriy -san　に確認して対応なしです。
- [ ] **Testing the locale resolution changed this time is mandatory.**
Why can we merge features that not tested yet?

Reply:
JUnit test code never maintained correctly before ever. I modified existing method but the test for that method is not exist.  I talked to @hirakuriy about this issue and he said you do not need to maintain JUnit test code.
Please talk to @hirakuriy and make decision about the maintenance policy, consider how to deal with currently not existing tests.

Reply:
@hirakuriy Leave it to you. But We must ensure the quality of the code alone.